### ISSUE TITLE
Question : Cannot test clone in Development Setup step

### ISSUE THREAD

**Issue by capsci** [exercism/go#1415]
I cloned the repo and _cd_'d into directory.

go > bin/test-without-stubs 
bin/test-without-stubs: line 47: bin/golangci-lint: No such file or directory
moving stub files back
mv: rename exercises/*/*.notgo to exercises/*/*.go: No such file or directory

Any idea what might be missing?
ps - I found this commit (2128ee038d28ef33eefb3e3b02ce4acfadef4a4c) which added `bin/golang-ci` to `.gitignore`

**Comment by brugnara** [comment]
Hi.

Those commands are used by GitHub Actions, see here:

- https://github.com/exercism/go/blob/main/.github/workflows/exercise-tests.yml

Last commands:

```
        run: |
          ./bin/fetch-golangci-lint
          ./bin/test-without-stubs
```

Should add the missing tool you are looking for: [fetch-golangci-lint](https://github.com/exercism/go/tree/main/bin), infact, does:

```
#!/bin/bash

curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b ./bin latest
```

**Comment by ekingery** [comment]
Thanks @brugnara! That's right - @capsci, if you want to run those tests, you'll need to run those `fetch-*` scripts from the root of the repo, which will place the expected binaries in the `bin` directory. I'm going to close this out, but let us know if you run into any further issues. 

As a heads up, I just fixed the `configlet` linting problems in #1420. We're still working on getting the rest of the practice exercises stubbed out. Until then, the `test-without-stubs` check will fail.
